app/code/Magento/GoogleAnalytics/Test/Unit/Block/GaTest.php

```diff
@@ -178,29 +150,30 @@ public function testOrderTrackingData()
             ],
             'currency' => 'USD'
         ];
-
+        echo json_encode($expectedResult);
```

Comment:
I think this `echo`, and the `echo` a few lines below was left here by mistake. Please remove them.